new/LIT7142Semon.xml

```diff
@@ -0,0 +1,67 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7142Semon" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ስምዖን፡ ዘዓምድ፡</title>
+                <title xml:lang="gez" type="normalized"  corresp="#t1">Sǝmʻon za-ʻāmd</title>
+                <title xml:lang="en" xml:id="t2">Spiritual homilies of Macarius the Egyptian</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MKr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Homily"/>
+                    <term key="Translation"/>
+                    <term key="MonasticLiterature"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+            <abstract>
+                <p>It is a collection of spiritual homilies attributed to Macarius the Egyptian or (as in Ethiopian tradition) to Simeon Stylites.</p>
+            </abstract>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MKr" when="2024-11-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7142Semon" passive="PRS11392Simeon"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7142Semon" passive="PRS6427Macarius"/>
+                </listRelation>
+            </div><!---->
```

Comment:
copy incipit as provided in the issue? ንዌጥን፡ በረድኤተ፡ እግዚአብሔር፡ አቡሁ፡ ለእግዚእነ፡ ወመድኃኒነ፡ ወአብራሄ፡  etc

Reply:
I believe the same text is in EMDA177

Reply:
oops sorry I now see that you already provided the link to EMDA177 (which is the same as EAP432-1-34, but we go with the HMML shelfmarks, EAP are all doublets https://github.com/BetaMasaheft/Documentation/issues/2523) 

our link is therefore https://betamasaheft.eu/manuscripts/EMDA177/main